src/components/CoverTypeQuestion/CoverTypeQuestion.tsx

```diff
@@ -1,35 +1,34 @@
-import React, { useContext } from 'react';
+import React from 'react';
+import { Nullable } from '@constructor-io/constructorio-client-javascript/lib/types';
 import QuestionTitle from '../QuestionTitle/QuestionTitle';
-import QuizContext from '../CioQuiz/context';
 import QuestionDescription from '../QuestionDescription/QuestionDescription';
 import { renderImages } from '../../utils';
+import { QuestionImages } from '../../types';
 
-export default function CoverTypeQuestion() {
-  const { state } = useContext(QuizContext);
-  let question;
+export interface ICoverTypeQuestionProps {
```

Comment:
Making it clearer what CoverQuestion props are and passing them as props instead of depending on context in case a user want to use the component without necessarily having to implement context for it to work